app/src/main/AndroidManifest.xml

```diff
@@ -84,7 +86,7 @@
       android:windowSoftInputMode="adjustResize" />
     <activity
       android:name=".app.profile.ProfileChooserActivity"
-      android:label="@string/profile_chooser_activity_label"
+      android:label="@string/profile_chooser_activity_title"
```

Comment:
Similar to earlier PR: here & elsewhere, should these be synced to each activity's title?